app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -991,7 +1064,7 @@ DeviceDelegateOculusVR::EndFrame(const bool aDiscard) {
   // Add main eye buffer layer
   const float fovX = vrapi_GetSystemPropertyFloat(&m.java, VRAPI_SYS_PROP_SUGGESTED_EYE_FOV_DEGREES_X);
   const float fovY = vrapi_GetSystemPropertyFloat(&m.java, VRAPI_SYS_PROP_SUGGESTED_EYE_FOV_DEGREES_Y);
-  const ovrMatrix4f projectionMatrix = ovrMatrix4f_CreateProjectionFov(fovX, fovY, 0.0f, 0.0f, m.near, m.far);
+  const ovrMatrix4f projectionMatrix = ovrMatrix4f_CreateProjectionFov(fovX, fovY, 0.0f, 0.0f, VRAPI_ZNEAR, 0.0f);
```

Comment:
Is this still required after using the same projection matrix in the API? https://github.com/MozillaReality/FirefoxReality/blob/a24d68f238c98837f50c322555abee3c1e101278/app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp#L1006

Reply:
Ok, the doc says that `An infinite projection matrix is preferred for rasterization because, except for things *right* up against the near plane`

We don't have things right up against the near plane so it should be ok